doc/integrator/upgrade_application.rst.tmpl

```diff
@@ -85,11 +85,11 @@ Upgrade the main schema:
 .. prompt:: bash
 
    docker-compose exec geoportal alembic --name=main \
-        --config=geoportal/alembic.ini upgrade head
+        --config=alembic.ini upgrade head
```

Comment:
It's the default file name then it can be completely removed :-)
```suggestion
        upgrade head
```

Reply:
ok good to know thanks